play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/header/Header.scala

```diff
@@ -62,6 +64,13 @@ case class Header(
       Some(LanguageToggle(defaults ++ x.linkMap))
     case None    => None
   }
+
+  val languageToggleServiceNav: Option[LanguageToggle] = inputLanguageToggle match {
+    case Some(x) =>
+      val defaults: SortedMap[Language, String] = SortedMap(EnServiceNavigation -> "", CyServiceNavigation -> "")
+      Some(LanguageToggle(defaults ++ x.linkMap))
+    case None    => None
+  }
```

Comment:
We need to prepare a `languageToggle` for Service Navigation, so this part is duplicated too. Mostly because different default values. Again probably could be combined in one method